components/mediators/datamapper/org.wso2.carbon.mediator.datamapper.engine/src/main/java/org/wso2/carbon/mediator/datamapper/engine/core/mapper/MappingHandler.java

```diff
@@ -60,7 +60,7 @@ public MappingHandler(MappingResource mappingResource, String inputType, String
         this.mappingResource = mappingResource;
     }
 
-    public String doMap(InputStream inputMsg, Map propertiesMap)
+    public String doMap(InputStream inputMsg, Map<String, Map<String, Object>> propertiesMap)
```

Comment:
Where is the java doc for this public API method change? Why a Map of Map is now passed to this method? These should be part of java docs.